### PR TITLE
Update BigQuery Read related documentation 

### PR DESCRIPTION
**Pull request by amaliujia**
Update BigQuery Read related documentation.

Once your pull request has been opened and assigned a number, please edit the
URL below, replacing `PULL_REQUEST_NUMBER` with the number of your pull request.

http://apache-beam-website-pull-requests.storage.googleapis.com/488/index.html

Finally, it will help us expedite review of your Pull Request if you tag
someone (e.g. @username) to look at it.

### CODE REVIEW

**Comment by amaliujia** [comment]
R: @kennknowles @apilloud

**Comment by amaliujia** [comment]
R: @kennknowles @apilloud

**Comment by amaliujia** [comment]
Thinking this is broken: http://apache-beam-website-pull-requests.storage.googleapis.com/488/index.html

**Comment by amaliujia** [comment]
Thinking this is broken: http://apache-beam-website-pull-requests.storage.googleapis.com/488/index.html

**Comment by apilloud** [comment]
retest this please

**Comment by amaliujia** [comment]
retest this please

File: src/documentation/dsls/sql/create-table.md [hunk 1]
```diff
@@ -95,8 +95,7 @@ LOCATION '[PROJECT_ID]:[DATASET].[TABLE]'
 
 ### Read Mode
 
-Not supported. BigQueryI/O is currently limited to write access only in Beam
-SQL.
+Beam supports reading simpleType and ARRAY\<simpleType\> columns from a BigQuery table.
```
Comment (kennknowles) [inline src/documentation/dsls/sql/create-table.md]: Make use of code font and be clear what is syntax versus normal words. Perhaps:

```
Beam SQL supports reading columns with simple types (`simpleType` above) and arrays of simple types.
```

**Comment by kennknowles** [comment]
@asfgit merge

**Pull request merged**